backend/celery_worker.py

```diff
@@ -302,6 +310,110 @@ def check_if_is_premium_user():
     return True
 
 
+@celery.task(name="process_assistant_task")
+def process_assistant_task(
+    input_in: str,
+    files_name: List[str],
+    current_user: dict,
+    notification_id=None,
+) -> None:
+
+    logger.debug(f"Input: {input}")
+    logger.debug(type(input))
+    _input = InputAssistant.model_validate(input_in)
+    # _input = InputAssistant(**json.loads(input))  # type: ignore
+    # _input = InputAssistant(json.dumps(_input))
+    _current_user = UserIdentity(**current_user)  # type: ignore
+    try:
+        files = []
+        supabase_client = get_supabase_client()
+
+        for file_name in files_name:
+            tmp_name = file_name.replace("/", "_")
+            base_file_name = os.path.basename(file_name)
+            _, file_extension = os.path.splitext(base_file_name)
+
+            with NamedTemporaryFile(suffix="_" + tmp_name, delete=False) as tmp_file:
+                res = supabase_client.storage.from_("quivr").download(file_name)
+                tmp_file.write(res)
+                tmp_file.flush()
+
+                file_instance = File(
```

Comment:
Why are we creating two `File` classes here 🤔 ? 

An [`UploadFile`](https://fastapi.tiangolo.com/reference/uploadfile/) is FastAPI class that **to receive files from the request.** I don't see the need for a class here